packages/core/src/components/image-overlay/image-overlay.template.ts

```diff
@@ -0,0 +1,10 @@
+import { html } from "lit-html";
+
+import { ImageOverlay } from '@dso-toolkit/sources';
+import { imageTemplate } from '../../../../css/src/components/image/image.template';
```

Comment:
Dit zit mij toch niet lekker.. Ik heb 't idee dat we dit al eens hebben geprobeerd om vervolgens een release te moeten terugtrekken omdat de afnemer last had van deze import. Ff dubbelchecken dus.

Reply:
Ik vermoed dat ik in de war ben met een andere situatie waarin een script een module uit `@dso-toolkit/sources` importeerde. Maar dit is een file die afnemers in principe niet eens zouden aanroepen (en ik hoop zelfs dat ie niet eens in de NPM package zit..).

Ik laat het voor nu, het zou namelijk wel heel fijn zijn als dit zou werken.